docs/developer-guide/contributing/reporting-bugs.md

```diff
@@ -1,17 +1,6 @@
 # Reporting Bugs
 
-If you think you've found a bug in ESLint, please [create a new issue](https://github.com/eslint/eslint/issues/new?body=**What%20version%20are%20you%20using%3F**%0A%0A**What%20did%20you%20do%3F**%0A%0A**What%20happened%3F**%0A%0A**What%20did%20you%20expect%20to%20happen%3F**%0A%0A) on GitHub. Be sure to include the following information:
-
-1. The version of ESLint you are using.
-1. What you did.
-1. What you expected to happen.
-1. What actually happened.
-
-In addition, if you're reporting a bug with a rule, be sure to include:
-
-* The actual source code that is causing the issue
-* Your configuration
-* The complete and actual output from ESLint (don't summarize, we need the raw output)
+If you think you've found a bug in ESLint, please [create a new issue](https://github.com/eslint/eslint/issues/new?body=**What%20version%20are%20you%20using%3F**%0A%0A**What%20did%20you%20do%3F**%0A%0A**What%20happened%3F**%0A%0A**What%20did%20you%20expect%20to%20happen%3F**%0A%0A) or a [pull request](/docs/developer-guide/contributing/pull-requests) on GitHub. Be sure to copy the questions from the [bug report template](https://github.com/eslint/eslint/blob/master/templates/bug-report.md).
```

Comment:
This link still has a `body=` querystring param on the new issue link (it's been removed from New Rule and Rule Change). Not sure if this is intentional.